at_lookup/README.md

```diff
@@ -3,48 +3,108 @@
 ### Now for a little internet optimism
 
 # at_lookup library
-The AtLookup Library is the low-level direct implementation of the @protocol verbs
 
-## Installation:
-To use this library in your app, first add it to your pubspec.yaml
-```  
+## Overview:
+
+The AtLookup Library is the low-level direct implementation of the @protocol verbs. The AtClient packages depends on
```

Comment:
we need not mention atclient here..similar to at_client there may be other packages dependent on this